### DISCUSSION TITLE
iOS first run

### DISCUSSION THREAD

**Discussion by Try** [Try/OpenGothic#522] [Show and tell]
I've added basic most support for iOS in engine. Build process is still hairy, yet it getting close:
![изображение](https://github.com/Try/OpenGothic/assets/122894/c6dd187a-f6b9-42e1-aab3-53e9b37c8e24)

Build
1. Generate xcode-project
`cmake -H. -B./out/xcode-ios -G Xcode -DCMAKE_SYSTEM_NAME=iOS`
2. Select `Gothic2Notr` as startup  project. You may need to specify development-team in `Signing & Capabilities` section.
3. Copy gothic2 game files via Finder (you should be able to see OpenGothic folder now):
![изображение](https://github.com/Try/OpenGothic/assets/122894/93a50e73-75e0-4603-b0b4-6f130952ddbc)

4. Run the game
Note: input is not yet done, so you wont be able to pass further than a main menu. So, at this point iOS installation, is only iusefull for development purposes.

UPD:
build process now simplified - 4 steps instead of 7

**Comment by Try** [comment]
iPhone7
Most of time crashes with startup timeout (shaders take too long to compile apparently) or with OOM. But when it works:
![изображение](https://github.com/Try/OpenGothic/assets/122894/7d038d62-049b-43e4-aa21-c28c70265810)

**Comment by Try** [comment]
After GDR update, now game runs into. driver crash: https://developer.apple.com/forums/thread/750478

**Comment by Try** [comment]
Filed feature request to google, for valid CMake support on android: https://issuetracker.google.com/issues/353730410
(need to have at least as good as iOS)

**Comment by Ed0n3** [comment]
What’s the state of the game on iOS? No updates for over a year?

**Reply by Try** [reply]
Hi,
Unfortunately I don't have much throughput to implement thing such as screen-controller / joystick. So here, as often - OS level support is done: rendering/shaders/game-logic do work. For the rest - contributions are always welcome.

**Reply by Ed0n3** [reply]
Okay, thanks for the update :)
I was just curios if someone is working on it, can't really contribute, I have no experience in iOS dev at all (also no Mac device to compile on)
